zwave_js/DOCS.md

```diff
@@ -78,11 +85,38 @@ You can also use sites like this one to generate the required data:
 
 <https://www.random.org/cgi-bin/randbyte?nbytes=16&format=h>
 
-Ensure you keep a backup of this key. If you have to rebuild your system and
-don't have a backup of this key, you won't be able to reconnect to any securely
+Ensure you keep a backup of these key. If you have to rebuild your system and
+don't have a backup of these keys, you won't be able to reconnect to any securely
```

Comment:
Maybe "communicate" instead of "reconnect"? The driver can still talk to the device, it's just that the secure commands will fail.